### PR TITLE
[2.0.y] Bump p9dsu-xml to 734a35ebe5af49dba57269ca26065d40b876c962

### PR DESCRIPTION
**Pull request by youhour**
Bump p9dsu-xml to 734a35ebe5af49dba57269ca26065d40b876c962 to pickup the following fixes:

734a35e - careyz@supermicro.com - 2019-04-05 - MEMD:VM keyword is not set in system vpd on P9DSU-C
e1a56eb - careyz@supermicro.com - 2019-03-28 - Checkstop in istep 14.1 (mss_memdiag) - dimm garded
218a77a - jimsmc - 2018-05-03 - Fix duplicate capps target

### CODE REVIEW

**Comment by sammj** [comment]
> bug: issue1414 issue1382

Are these issues from the op-build repo or somewhere else?

**Comment by youhour** [comment]
> 
> 
> > bug: issue1414 issue1382
> 
> Are these issues from the op-build repo or somewhere else?

These are the github issues.  You can find them here https://github.com/open-power/boston-openpower/issues

**Comment by sammj** [comment]
That's a private repo so most people can't see that issue. Please update the pull-request with a description of the issues.

**Comment by youhour** [comment]
> 
> 
> That's a private repo so most people can't see that issue. Please update the pull-request with a description of the issues.

Let me see if there is an easy way to update the description.

**Comment by stewart-ibm** [comment]
Is this backporting a commit from master? If so, please ensure that commit is mentioned (i.e. `git cherry-pick -xs` or use stable tools to have "[abcdef1235 upstream]" at top of commit message).

I *really* don't want us to have references to private bug trackers is our commits. Doing so tends to make people lazy in describing the problem and the solution in the commit message.

> Pickup the latest p9sdu-xml changes

"latest" is subjective and will be incorrect in the future. Better to say "Bump p9dsu-xml to 734a35ebe5af"

> Fix issue 1414:  MEMD:VM keyword is not set in system vpd on P9DSU-C

Don't mention issue number, one would assume it's an op-build issue number, which it isn't.

This isn't the best description of the problem/solution either, but I'm guessing the p9dsu-xml commit isn't any better either.

> Fix issue 1381:  Checkstop in istep 14.1 (mss_memdiag) - dimm garded

Again, don't mention issue number.
What changes to avoid this?

> Fix duplicate capps target

okay. I'm not sure of the impact of this, but that's not a big concern.

> bug: issue1414 issue1381

The correct format for pointing to bugs that are fixed is using the "Fixes: URL/bug-number" notation, one per line.

> branch:  op-build-2.0.y

This is not needed. If the commit is being cherry-picked back from master (which I'm guessing it should be, unless these bugs are *specific* to 2.0.y and 2.3.y, then have the "cherry-picked from abcd1234" format as provided by `git cherry-pick -xs`.

**Comment by youhour** [comment]
@stewart-ibm @sammj Any thing else to change to this PR in order to process it?

**Comment by stewart-ibm** [comment]
> @stewart-ibm @sammj Any thing else to change to this PR in order to process it?

You've only changed the PR, not the commit. So yes, the changes still need to be made to the commit.

**Comment by youhour** [comment]
> 
> 
> > @stewart-ibm @sammj Any thing else to change to this PR in order to process it?
> 
> You've only changed the PR, not the commit. So yes, the changes still need to be made to the commit.

Done.  Hopefully this is good.  Let me know if there is another other thing to correct.  Thanks for helping.

File: openpower/configs/p9dsu_defconfig [hunk 1]
```diff
@@ -44,7 +44,7 @@ BR2_OPENPOWER_PLATFORM=y
 BR2_OPENPOWER_POWER9=y
 BR2_HOSTBOOT_CONFIG_FILE="p9dsu.config"
 BR2_OPENPOWER_MACHINE_XML_GITHUB_PROJECT_VALUE="p9dsu-xml"
-BR2_OPENPOWER_MACHINE_XML_VERSION="32ce616ef80f5cc3f128b3aa08fe50954a087888"
```
Comment (stewart-ibm) [inline openpower/configs/p9dsu_defconfig]: This is what's currently in op-build master, and the bump to 734a35ebe5af49dba has *not* even been in a pull request for op-build master yet.

Please ensure that this change is merged to master *first*, and then cherry-picked back to 2.0.y and 2.3.y branches. If for some reason (and I can't think of any) these do not apply for master, please explain *why* that's the case in the commit message here.

Comment (youhour) [reply openpower/configs/p9dsu_defconfig]: @stewart-ibm 
1)  There is no plan to support Boston in the master branch at this time.
2)  I have noticed that beside BR2_OPENPOWER_MACHINE_XML_VERSION variable change, the xml file is different between master branch and 2.0.y and 2.3.y.   I don't feel like we need to make this move to a shipped firmware.

diff 2.0.y  master
3d2
< BR2_PACKAGE_HOST_LINUX_HEADERS_CUSTOM_4_19=y
14a14
> BR2_ROOTFS_USERS_TABLES="$(BR2_EXTERNAL_OP_BUILD_PATH)/configs/users-table"
19c19
< BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.19.30"
---
> BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="5.1.4"
40a41,42
> BR2_PACKAGE_SUDO=y
> BR2_PACKAGE_UTIL_LINUX_AGETTY=y

diff 2.3.y master
19c19
< BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="5.0.7"
---
> BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="5.1.4"

It appears that kernel 5.1.4 is not merged to 2.3.y and therefore 2.3.y cannot use xml file from master.

Comment (stewart-ibm) [reply openpower/configs/p9dsu_defconfig]: If Boston cannot be supported, then we need to:
1) Remove it from master
2) find replacement machines for the Bostons we have and rely on for our work and regression testing.

Comment (stewart-ibm) [reply openpower/configs/p9dsu_defconfig]: You also do not need the same kernel version in 2.3.y and master, you just need to resolve any conflicts.

Comment (stewart-ibm) [reply openpower/configs/p9dsu_defconfig]: Also, as per https://open-power.github.io/op-build/process/KernelTree.html#versioning op-build 2.0.y will *never* get any kernel later than a 4.19 kernel (it's a longterm kernel).

What *exactly* is the dependency between kernel version and the p9dsu-xml change? I can't see how there could possibly be one.

**Comment by stewart-ibm** [comment]
Is this PR going anywhere?

I haven't seen PRs to master, so if there's no response in the next couple of days, I'll close this PR.

**Comment by stewart-ibm** [comment]
> Is this PR going anywhere?
> 
> I haven't seen PRs to master, so if there's no response in the next couple of days, I'll close this PR.

I gather this PR is no longer needed, closing.

**Pull request closed without merging**